openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -64,6 +69,30 @@ describe("OpenAssessment.ResponseView", function() {
             return this.removeFileError ? errorPromise : successPromise;;
         }
 
+        this.teamListError = false
+        this.listTeamsResult = {name: 'TeamName', id:'TeamId'}
+        this.listTeams = function(courseId) {
+            if (this.teamListError) {
+                return errorPromise
+            } else {
+                return successPromiseWithResult(this.listTeamsResult)
+            };
+        }
+        
+        this.teamDetailError = false
+        this.teamDetailResult = {
+            course_id: 'CourseID',
+            description: "A team!",
+            membership: [{user: 'user1'}, {user: 'user2'}, {user: 'user3'}]
```

Comment:
What is the benefit of having a key in a dictionary with a single value? Would something like: membership:` ['user1', 'user2','user3']` make more sense?

Reply:
This is a mock of what we will recieve from the teams API. There are more keys here but currently I'm only using the one so that's all I mocked